python/caffe/_caffe.cpp

```diff
@@ -268,8 +268,11 @@ BOOST_PYTHON_MODULE(_caffe) {
     bp::no_init)
     .def("__init__", bp::make_constructor(&Net_Init))
     .def("__init__", bp::make_constructor(&Net_Init_Load))
-    .def("_forward", &Net<Dtype>::ForwardFromTo)
-    .def("_backward", &Net<Dtype>::BackwardFromTo)
+    // Instead of ForwardFromTo and BackwardFromTo,
+    // expose PyForwardFromTo and PyBackwardFromTo,
+    // basically the same thing but with GIL release
```

Comment:
The word "basically" is vague.  A better explaining comment would be:
```
// Instead of ForwardFromTo and BackwardFromTo,
// expose PyForwardFromTo and PyBackwardFromTo,
// which adds a GIL release around these functions
```